analytics/src/main/java/brocoeur/example/analytics/controller/UserWinLossByGameController.java

```diff
@@ -31,8 +31,11 @@ public void saveUsers() {
         List<UserWinLossByGame> userWinLossByGameServices = new ArrayList<>();
         userWinLossByGameServices.add(new UserWinLossByGame(123, 5, "Roulette", "MeatAlive", 0, 0));
         userWinLossByGameServices.add(new UserWinLossByGame(324, 5, "Coin Toss", "MeatAlive", 0, 0));
+        userWinLossByGameServices.add(new UserWinLossByGame(666, 5, "Coin Toss", "MeatAlive", 0, 0));
```

Comment:
666 should be the game id and this is linked to the game name. in this case, 666 is wrong with Coin Toss. To change

Reply:
misstype fixed